-------------------------------------------------------------------------------
Test set: org.cis120.Task5Test
-------------------------------------------------------------------------------
Tests run: 3, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 0 s <<< FAILURE! - in org.cis120.Task5Test
testKickOneChannel  Time elapsed: 0 s  <<< ERROR!
java.lang.NullPointerException: Cannot invoke "java.util.Collection.contains(Object)" because "existingUsers" is null
	at org.cis120.Task5Test.setUp(Task5Test.java:35)

testInviteByOwner  Time elapsed: 0 s  <<< ERROR!
java.lang.NullPointerException: Cannot invoke "java.util.Collection.contains(Object)" because "existingUsers" is null
	at org.cis120.Task5Test.setUp(Task5Test.java:35)

testInviteByNonOwner  Time elapsed: 0 s  <<< ERROR!
java.lang.NullPointerException: Cannot invoke "java.util.Collection.contains(Object)" because "existingUsers" is null
	at org.cis120.Task5Test.setUp(Task5Test.java:35)

